ionic_user_interface/src/views/Signup.vue

```diff
@@ -70,6 +70,19 @@
                 >
                   Sign up
                 </ion-button>
+                <div class="privacy-policy">
+                  <ion-checkbox v-model="termsCheckbox" />
+                  <ion-text>
+                    I agree to
+                    <a href="https://routemaker.rocks/privacy.html" target="_blank">
+                      Privacy Policy
```

Comment:
![image](https://user-images.githubusercontent.com/46164638/126193305-0c6a677a-49e8-4421-8da6-63970717f680.png)

The underline is a little weird, maybe you can try changing `&#38;` to `&nbsp;&#38;&nbsp;`